Src/DependencyCollector/Net46.Tests/DependencyTrackingTelemetryModuleTestNet46.cs

```diff
@@ -108,7 +120,7 @@ public void TestDependencyCollectionEventSource()
 
         [TestMethod]
         [Timeout(5000)]
-        public void TestDependencyCollectionEventSource404()
+        public async Task TestDependencyCollectionEventSource404()
```

Comment:
Can you remove 404 from the test name and replace with just 'failed' - I used 404 earlier as i google.com/404 was the only reliable http call which returned non-sucess status code. Now that you control the 'server', you can return any fail status code. Removing 404 and naming it just 'fail' clears any confusion people may have in future about 404 being special in someway..